backend/svirt.pm

```diff
@@ -84,6 +84,9 @@ sub do_stop_vm {
             my $ps = 'powershell -Command';
             $self->run_ssh_cmd("$ps Stop-VM -Force -VMName $vmname -TurnOff");
             $self->run_ssh_cmd(qq($ps "\$ProgressPreference='SilentlyContinue'; Remove-VM -Force -VMName $vmname"));
+            my $vhdx_path = "${\get_var('HYPERV_DISK', 'D:')}${\get_var('HYPERV_ROOT', '')}\\\\cache\\$vmname";
```

Comment:
nitty...
```suggestion
            my $vhdx_path = get_var('HYPERV_DISK', 'D:') . get_var('HYPERV_ROOT', '') . '\\cache\' . $vmname;
```

Reply:
when it gets complicated, I am a fan of `sprintf`:
```suggestion
            my $vhdx_path = sprintf "%s%s\\\\cache\\%s", get_var('HYPERV_DISK', 'D:'), get_var('HYPERV_ROOT', ''), $vmname;
```

Reply:
Works nicely, thanks!